metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -1009,6 +1009,48 @@ def test_special_liquid_variables():
     assert actual_text == expected_text
 
 
+@freeze_time(FROZEN_TIME)
+def test_incremental_liquid_expression():
+    text: str = """{% assign source_table_variable = "source_table" | sql_quote | non_existing_filter_where_it_should_not_fail %}
```

Comment:
can we make this test more narrowly targeted to testing `incrementcondition`? or is there a reason to test all of the jinja operators like this in one go?